api/client/client.go

```diff
@@ -3337,8 +3337,8 @@ func (c *Client) ListResources(ctx context.Context, req proto.ListResourcesReque
 			resources[i] = respResource.GetKubernetesServer()
 		case types.KindUserGroup:
 			resources[i] = respResource.GetUserGroup()
-		case types.KindAppOrSAMLIdPServiceProvider:
-			resources[i] = respResource.GetAppServerOrSAMLIdPServiceProvider()
+		case types.KindSAMLIdPServiceProvider:
+			resources[i] = respResource.GetSAMLIdPServiceProvider()
```

Comment:
This causes the following error in Connect if I call `GetApps` (for example, in access requests table) against the cluster that still returns the "old" `types.KindAppOrSAMLIdPServiceProvider`:
```
12 UNIMPLEMENTED: resource type app_server_or_saml_idp_sp does not support pagination
```

Shouldn't we still support it, since this kind is deprecated, not removed?

Reply:
Should be resolved in latest changes. Can you please pull it and try again. Note that Connect should be tested with updated `tsh` built from this PR.